test/functional/tests/performance/test_performance_write.py

```diff
@@ -0,0 +1,165 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# Copyright(c) 2024 Huawei Technologies Co., Ltd.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+import pytest
+
+from datetime import timedelta
+from api.cas import casadm
+from api.cas.cache_config import CacheLineSize, CacheMode, SeqCutOffPolicy
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import ReadWrite, IoEngine, CpusAllowedPolicy
+from test_utils.os_utils import (
+    set_wbt_lat,
+    get_number_of_processors_from_cpuinfo,
+    get_dut_cpu_physical_cores,
+)
+from test_utils.size import Unit, Size
+
+
+# TODO: for disks other than Intel Optane, fio ramp is needed before fio tests on raw disk
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+@pytest.mark.parametrizex(
+    "block_size", [Size(1, Unit.Blocks4096), Size(8, Unit.Blocks4096)]
+)
+@pytest.mark.parametrizex("queue_depth", [1, 16, 32])
+def test_performance_write_hit_wb(block_size, queue_depth):
+    """
+    title: Test Open CAS performance for 100% write inserts scenario in write-back mode.
```

Comment:
The title says it's write inserts test (which is consistent with the implementation), but the name says "write_hit_wb". I suggest to change the name to "write_insert_wb".